sklearn/neighbors/tests/test_kde.py

```diff
@@ -141,3 +141,55 @@ def test_kde_pipeline_gridsearch():
     search = GridSearchCV(pipe1, param_grid=params, cv=5)
     search.fit(X)
     assert_equal(search.best_params_['kerneldensity__bandwidth'], .1)
+
+
+def test_kde_sample_weights():
+    n_samples = 2500
```

Comment:
How long does this test take to run? This seems larger than necessary to prove the point

Reply:
5 seconds on my laptop. Indeed, I can decrease this to a few 100s and this would take ~1 sec.